src/SamplesApp/UITests.Shared/Toolkit/ElevatedView_CornerRadius.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Threading.Tasks;
+using Windows.UI.Xaml.Controls;
+using Uno.UI.Samples.Controls;
+
+namespace UITests.Toolkit
+{
+	[Sample]
+	public sealed partial class ElevatedView_CornerRadius : Page
+	{
+		public ElevatedView_CornerRadius()
+		{
+			this.InitializeComponent();
+
+			Do();
+		}
+
+		private async void Do()
+		{
+			await Task.Delay(200);
+			var tree = this.ShowLocalVisualTree(0);
```

Comment:
Codacy found an issue: [Remove the unused local variable 'tree'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6254623)